src/sdk/propagation/baggage.rs

```diff
@@ -13,7 +13,7 @@
 //! # Examples
 //!
 //! ```
-//! use opentelemetry::api::{BaggageExt, Key, propagation::TextMapPropagator};
+//! use opentelemetry::api::{Key, propagation::TextMapPropagator, BaggageExt};
```

Comment:
I think this can be reverted (rustfmt would have them sorted alphabetically)

Reply:
Must be something I changed after running rustfmt. Should really have enabled auto format on save 😞

Reply:
Oh rustfmt can't reformat code in doc comments yet unfortunately (also I think it would actually have modules first, so maybe move Key to the end instead?)